src/sdk/namespace_fs.js

```diff
@@ -1276,6 +1331,17 @@ class NamespaceFS {
         }
     }
 
+
+    async _delete_single_object(fs_context, file_path, params) {
+        try {
+            await nb_native().fs.unlink(fs_context, file_path);
+        } catch (err) {
+            if (err.code !== 'ENOENT') throw err;
+        }
+        await this._delete_path_dirs(file_path, fs_context);
+        if (this._is_directory_content(file_path, params.key)) await this._clear_user_xattr(fs_context, params);
```

Comment:
move the explanation from the function to here

Reply:
done